README.md

```diff
@@ -14,9 +14,19 @@ Online documentation is available at https://mozilla.github.io/mozanalysis/
 pip install mozanalysis
 ```
 
+## Local Installation
+```bash
+# Create and activate a python virtual environment.
+python3 -m venv venv/
+source venv/bin/activate
+pip install -r requirements.txt
+```
+
 ## Development
 Linting and Formatting are done with Ruff.
 
+Update the `requirements.txt` file using the script `script/update_deps`
+
```

Comment:
What's the actual procedure to update a dependency? Do you manually install the dependency into your environment and then run `update_deps`?

Reply:
Added to the README

Reply:
Good thing I asked lol